### PR TITLE
docs(guides) updates due to recent plugin update

### PR DESCRIPTION
**Pull request by EugeneHlushko**
As a follow up to recent clean-webpack-plugin 's update, we want to reflect all the necessary changes

Related to #2884

### CODE REVIEW

**Comment by montogeek** [comment]
Use same version?
image

**Comment by EugeneHlushko** [comment]
How in a world did that happen

File: src/content/guides/development.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ __package.json__
     "author": "",
     "license": "ISC",
     "devDependencies": {
-      "clean-webpack-plugin": "^0.1.16",
+      "clean-webpack-plugin": "^2.0.1",
```
Comment (EugeneHlushko) [inline src/content/guides/development.md]:
```suggestion
      "clean-webpack-plugin": "^2.0.0",
```

**Comment by EugeneHlushko** [comment]
@montogeek its there now

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**